mixins/labelled-mixin.js

```diff
@@ -27,6 +27,24 @@ const getLabel = labelElem => {
 	else return labelElem.textContent;
 };
 
+const waitForElement = async(contextElement, selector, timeout) => {
```

Comment:
I _think_ with some fiddling this function could be made smaller by making it recursive... but not that important.